_data/collection-index.yml

```
@@ -268,6 +268,11 @@
   contact: https://github.com/ijnek/ros-devcontainer-template/issues
   repository: https://github.com/ijnek/ros-devcontainer-template
   ociReference: ghcr.io/ijnek/ros-devcontainer-template
+- name: ROS 2 Workspace Templates
+  maintainer: Bruno-Pier Busque
+  contact: https://github.com/BrunoB81HK/ros2-workspace-devcontainer-template/issues
+  repository: https://github.com/BrunoB81HK/ros2-workspace-devcontainer-template
+  ociReference: https://ghcr.io/BrunoB81HK/ros2-workspace-devcontainer-template
```

```suggestion
  ociReference: ghcr.io/brunob81hk/ros2-workspace-devcontainer-template
```

Hi 👋

The ORAS spec requires the ociReference to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.